src/Core/Content/Media/Thumbnail/ThumbnailService.php

```diff
@@ -239,6 +239,20 @@ private function getImageResource(MediaEntity $media)
             throw new FileTypeNotSupportedException($media->getId());
         }
 
+        try {
+            $exif = exif_read_data($filePath);
```

Comment:
This will explode when the customer does not have installed the php exif extension. It's also not required by Shopware 6

Reply:
@shyim Would it be better with the check if exif extension is enabled?

Reply:
Why not function_exists?

Reply:
If you like that better i will change it. ;)